### PR TITLE
feat: news and gov news rss sync page

### PR DESCRIPTION
**Pull request by johnhorsema**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] i18n translation has been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by nandiheath — COMMENTED** [review]
is it possible to fetch en news as well?

File: gatsby-config.js [hunk 1]
```diff
@@ -45,14 +45,14 @@ module.exports = {
...
     },
-    `gatsby-plugin-react-helmet`,
     {
       resolve: `gatsby-source-filesystem`,
       options: {
         name: `images`,
         path: `${__dirname}/src/images`,
       },
     },
+    `gatsby-plugin-react-helmet`,
```
Comment (nandiheath) [inline gatsby-config.js:55]: why changine the order?

File: src/pages/news.js [hunk 2]
```diff
@@ -1,27 +1,74 @@
...
-const LatestMediaNewsPage = ({ data, pageContext }) => {
+const NewsPage = props => {
+  const { data } = props
   const { t } = useTranslation()
+
+  const item = ({ node }) => {
+    return (
+      <NewsCard>
+        <Typography variant="body2" color="textPrimary">
+          {node.source} {node.date} {node.time}
```
Comment (nandiheath) [inline src/pages/news.js]: prefer to use template ````

File: src/pages/news.js [hunk 3]
```diff
@@ -1,27 +1,74 @@
...
+        <Link
+          href="https://news.google.com/topics/CAAqJQgKIh9DQkFTRVFvSUwyMHZNR1J4T1hBU0JYcG9MVWhMS0FBUAE?hl=zh"
+          target="_blank"
+        >
+          {t("news.source_google")}
+        </Link>
+      </Typography>
+      <Typography variant="body2">
+        {t("waiting_time.last_updated")}
+        {data.allGoogleNews.edges[0].node.date} {data.allGoogleNews.edges[0].node.time}
```
Comment (nandiheath) [inline src/pages/news.js]: use template as well
and use lodash.get to avoid undefined

**Review by nandiheath — CHANGES_REQUESTED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (johnhorsema) [reply gatsby-config.js:55]: Oh typo. Will revert.

**Comment by johnhorsema** [comment]
@nandiheath it is possible to fetch en news just change lh=en in the url. But I don't know how to make it work in static site

File: src/pages/news.js (hunk 2 above)
Comment (johnhorsema) [reply src/pages/news.js]: Not understand. What template?

File: src/pages/news.js (hunk 3 above)
Comment (johnhorsema) [reply src/pages/news.js]: where to put the lodash.get?

**Review by nandiheath — APPROVED** [review]
LGTM

**Pull request merged**